hbase-rsgroup/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupAdminEndpoint.java

```diff
@@ -508,7 +508,7 @@ public void preCreateTableAction(
     if (desc.getTableName().isSystemTable()) {
       return;
     }
-    RSGroupInfo rsGroupInfo = groupInfoManager.determineRSGroupInfoForTable(desc.getTableName());
+    RSGroupInfo rsGroupInfo = determineRSGroupInfoForTable(desc);
```

Comment:
We should handle the table modify scenario as well when RSGroup is modified.

Reply:
Handled it.